src/gigs-board/components/posts/PostEditor.jsx

```diff
@@ -79,6 +83,10 @@ initState({
   githubLink: props.githubLink ?? "",
   warning: "",
 });
+const savedState = localStorage.getItem("widgetState");
```

Comment:
`localStorage.getItem` is not going to work, need `Stroage.get`

Reply:
updated will push to all Carina's branch shortly